### DISCUSSION TITLE
What if I want to cancel Protonmail and keep SimpleLogin?

### DISCUSSION THREAD

**Discussion by pedrom34** [simple-login/app#2212] [Q&A] [answered]
Hi,

I'm a paid proton user and use SimpleLogin a lot. What happen if I cancel my paid plan with proton but want to keep simplelogin and keep all my created addresses and custom domains?
I mean: what's the process? Should I cancel proton first?

Thanks!

**Comment by nguyenkims** [comment, marked as answer]
Hi in that case you can subscribe to SL premium. 

For info this is extracted from our pricing page

> When your subscription ends, all aliases you created continue working normally, both on receiving and sending emails. Concretely:
> - All aliases/domains/directories/mailboxes you have created are kept and continue working normally.
> - You cannot create new aliases if you exceed the free plan limit, i.e. have more than 10 aliases.
> - As features like catch-all or directory allow you to create aliases on-the-fly, those aliases cannot be automatically created if you have more than 10 aliases.
> - You cannot add new domain, directory or mailbox.
> 
> For example, if you have 100 aliases by the time your subscription ends, these 100 aliases will continue receiving and sending emails normally. You cannot however create new aliases.

**Reply by pedrom34** [reply]
Thanks !

**Comment by jenkisan** [comment]
Another option, just saying, is use a custom domain.  This way should anything happen even to Simplelogin you have the domain and will keep receiving all emails from that domain just in case.
